### ISSUE TITLE
Result styling

### ISSUE THREAD

**Issue by Artoria2e5** [tranleduy2000/ncalc#5]
The current implementation centers the result and renders in some Math-y serif font, which is not necessarily the most readable thing on screen. Ncalc should consider:

* Right-aligning the result
* Using a sans-serif for result (https://tex.stackexchange.com/questions/77932/the-right-way-to-get-sans-serif-math)

Additional issues include size and line-breaking, both of which appear not quite easy to address for now.

**Comment by Madis0** [comment]
Agreed, it looks very ugly and out of place (for simple equations at least). 
Maybe provide a font option for that too?

**Comment by Artoria2e5** [comment]
PerKhan/KaTeX#123 suggests that this thing can be hard-done by finding a font and changing the metrics. No switching feature is available it seems.
